app/controllers/problems_controller.rb

```diff
@@ -61,7 +61,7 @@ def show
   end
 
   def create_issue
-    body = render_to_string "issue_trackers/issue", layout: false, formats: [:txt]
+    body = render_to_string "issue_trackers/issue", layout: false, formats: [:md]
```

Comment:
what I want in the future is,probably, have the issue tracker doing a POST to this endpoint passing the right format, so we can render with different formats depending on what the issue trakcer supports.